### PR TITLE
updated analyzer for fake rate study for resolved top tagger

### PR DESCRIPTION
**Pull request by semrat**

### CODE REVIEW

File: Analyzer/include/Config.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include "Framework/Framework/include/PartialUnBlinding.h"
 #include "Framework/Framework/include/StopGenMatch.h"
 #include "Framework/Framework/include/MegaJetCombine.h"
+#include "Framework/Framework/include/MT2Hemispheres.h"
```
Comment (cmadrid1) [inline Analyzer/include/Config.h]: I cant merge this with the MT2 stuff here

File: Analyzer/include/Config.h [hunk 2]
```diff
@@ -217,6 +219,27 @@ class Config
             };
             registerModules(tr, std::move(modulesList));
         }
+        else if (analyzer=="MakeMT2Hemispheres")
```
Comment (cmadrid1) [inline Analyzer/include/Config.h]: I cant merge this with the MT2 stuff here

**Comment by cmadrid1** [comment]
Just keep working on adding the MT2 stuff since you already started and when it is ready I'll merge this all in.

File: Analyzer/include/Config.h (hunk 1 above)
Comment (semrat) [reply Analyzer/include/Config.h]: I can not understand ! why you can not, but anyway I will be keeping going to update MT2.

Comment (cmadrid1) [reply Analyzer/include/Config.h]: The code will not compile because this file doesn't exist in the Framework repo.

Comment (semrat) [reply Analyzer/include/Config.h]: Yeap I realized :) ok I am updating again

**Pull request merged**